xray/commands/offlineupdate/offlineupdate.go

```diff
@@ -110,11 +113,27 @@ func getURLsToDownloadDBSyncV3(responseBody []byte, isPeriodicUpdate bool) ([]st
 		}
 		for _, packageUrl := range onboardingResponse {
 			urlsToDownload = append(urlsToDownload, packageUrl.DownloadUrl)
+			logV3UpdateFileChecksum(packageUrl)
 		}
 	}
 	return urlsToDownload, nil
 }
 
+func logV3UpdateFileChecksum(updateFileItem V3UpdateResponseItem) {
```

Comment:
Let's add a unit test for this function in `offlineupdate_test.go`. Let's follow the table-testing pattern. See other tests in that file as a reference.

Reply:
Done